### PR TITLE
Added support for integers inside expressions

### PR DESCRIPTION
**Pull request by ruudk**
See #182

### CODE REVIEW

File: Security/Authorization/Expression/ExpressionParser.php [hunk 1]
```diff
@@ -105,6 +105,10 @@ private function Primary()
             return new ConstantExpression($this->match(ExpressionLexer::T_STRING));
         }
 
+        if ($this->lexer->isNext(ExpressionLexer::T_INTEGER)) {
+            return new ConstantExpression((int) $this->match(ExpressionLexer::T_INTEGER));
```
Comment (stof) [inline Security/Authorization/Expression/ExpressionParser.php]: casting the value to an integer should rather be done in the lexer, to be consistent with the handling of strings

Comment (ruudk) [inline Security/Authorization/Expression/ExpressionParser.php]: It already felt weird :) Will fix it.

Comment (ruudk) [inline Security/Authorization/Expression/ExpressionParser.php]: Done!

**Comment by stof** [comment]
:+1:

**Pull request merged**